src/usr/local/www/services_captiveportal.php

```diff
@@ -278,6 +286,10 @@
 		}
 	}
 
+	if (isset($_POST['noconcurrentlogins']) && !in_array($_POST['noconcurrentlogins'], array('disabled', 'multiple', 'last', 'first'))) {
```

Comment:
See other note about using the array keys of the option list variable for this instead of hardcoding the possible choices in an array.